src/components/seeds/database/index.tsx

```diff
@@ -642,29 +642,33 @@ export default function Database(props: DatabaseProps): JSX.Element {
                         sx={{ background: '#F2F4F5', display: 'flex', color: '#000000', padding: 2, flexWrap: 'wrap' }}
                       >
                         <Typography>{strings.TOTAL}</Typography>
```

Comment:
Why is total outside the box?

Reply:
Total <count, label> <count, label> <count, label>
The count/label pairs are in boxes because they need to be displayed together without any wrapping.

Reply:
Oh, ok. I thought "Total <num> accessions" was one thing